docs/BindingsGuide_Dotnet.md

```diff
@@ -1,5 +1,23 @@
 # Azure SQL bindings for Azure Functions - .NET
 
+## Setup Function App
+1. Install [Azure Functions Core Tools](https://docs.microsoft.com/azure/azure-functions/functions-run-local)
```

Comment:
Let's be explicit that this is a one-time setup (do similar for the other pages)

```suggestion
## Setup Function App

These instructions will guide you through creating your Function App and adding the SQL binding extension. This only needs to be done once for every function app you create.

1. Install [Azure Functions Core Tools](https://docs.microsoft.com/azure/azure-functions/functions-run-local)
```